### PR TITLE
Refactor: Remove memoization from spec test

### PR DESCRIPTION
**Pull request by RobBoothAppDev21**
Removed Memoization (let, before, etc.) from specs:
- spec/helpers/url_helpers.rb
- spec/integrations/feed_importing_spec.rb
- spec/models/story_spec.rb
- spec/requests/feeds_controller_spec.rb
- spec/requests/imports_controller_spec.rb

### CODE REVIEW

**Comment by RobBoothAppDev21** [comment]
@mockdeep I updated the pr can you take a look at ?

File: spec/models/story_spec.rb [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # frozen_string_literal: true
 
 RSpec.describe "Story" do
-  let(:story) { build_stubbed(:story) }
+  # let(:story) { build_stubbed(:story) }
```
Comment (mockdeep) [inline spec/models/story_spec.rb:4]: I think this comment can be removed.

File: spec/helpers/url_helpers_spec.rb [hunk 2]
```diff
@@ -1,7 +1,7 @@
 # frozen_string_literal: true
 
 RSpec.describe UrlHelpers do
-  let(:helper) do
+  def create_url_helper
```
Comment (mockdeep) [inline spec/helpers/url_helpers_spec.rb:4]: I think we could probably still call this `helper`.

```suggestion
  def helper
```

File: .rubocop.yml [hunk 3]
```diff
@@ -25,6 +25,7 @@ Metrics/BlockLength: { Exclude: [config/**/*.rb, spec/**/*.rb, db/seeds/**/*.rb]
 Metrics/MethodLength: { Exclude: [db/migrate/*.rb] }
 Rails/SkipsModelValidations: { AllowedMethods: [update_all] }
 RSpec/DescribeClass: { Exclude: [spec/system/**/*] }
+RSpec/ExampleLength: { Exclude: [spec/integration/feed_importing_spec.rb] }
```
Comment (mockdeep) [inline .rubocop.yml:28]: For exceptions like this, I would add them to `.rubocop_todo.yml` instead. I generally put permanent exceptions here, but the todo file is for stuff we hope to improve at some point.

**Review by mockdeep — COMMENTED** [review]
Looking pretty good! I left some minor comments.

**Comment by RobBoothAppDev21** [comment]
@mockdeep updated pr

**Comment by mockdeep** [comment]
Looks great!

**Pull request merged**